app/views/posts/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<main class="card">
+  <h1>Add a new post</h1>
+  <%= form_with model: @post, url: user_posts_path, class: "form" do |form| %>
+    <%= form.label :title %> <br>
+    <%= form.text_field :title %> <br>
+    <%= form.label :text %> <br>
+    <%= form.text_area :text , cols: 30, rows: 10 %> <br>
+    <%= form.submit "Submit", class: "btn" %>
+  <% end %>
+</main>
```

Comment:
- [ ] @yemidada Great work adding the form to add a new post, kindly check your controller methods and how you configure the post for to ensure they are working correctly.(the add comments and like post are working correctly)

Reply:
@danmainah I dont your feedback